Sources/SwiftIDEUtils/SyntaxClassifier.swift

```diff
@@ -196,12 +196,23 @@ private struct ClassificationVisitor {
 
     for case (let index, let child?) in children.enumerated() {
 
-      let classification: (SyntaxClassification, Bool)?
+      let classification: (classification: SyntaxClassification, force: Bool)?
       if case .layout(let layout) = descriptor.node.kind.syntaxNodeType.structure {
         classification = SyntaxClassification.classify(layout[index])
       } else {
         classification = nil
       }
+
+      if let classification, classification.force == true {
+        let range = SyntaxClassifiedRange(
+          kind: classification.classification,
+          range: ByteSourceRange(offset: byteOffset, length: child.byteLength - child.leadingTriviaByteLength - child.trailingTriviaByteLength)
+        )
+        report(range: range)
+        byteOffset += child.byteLength
+        continue
+      }
+
```

Comment:
Oh, did you find a solution to https://github.com/apple/swift-syntax/pull/1953#discussion_r1276769666 after all? Or doesn’t this work?

Reply:
That could be a solution if we don't refactoring `ClassificationVisitor`,  and that works.😉

Reply:
I think that’s a fine solution. Good idea 👍🏽